INSTALL.rst

```diff
@@ -34,7 +113,8 @@ We build assets using `npm`. Make sure you have installed it system wide.
 
 .. code-block:: console
 
-    (inspirehep)$ npm update && npm install -g node-sass clean-css requirejs uglify-js
+    (inspirehep)$ sudo npm update
+    (inspirehep)$ sudo npm install -g node-sass clean-css requirejs uglify-js
```

Comment:
That does not look really like 'fixing', it's changing the ownership of a system directory... though the second option there (per-user "global" npm directory is a nice option).

In any case, this is meant to work on a fresh install, with the default permissions. I can add a note pointing to that page if you want. Will have to check if the solution is generic or macos bound tohugh. Btw. there's already a todo message to add ubuntu and macos examples too, might be a subtask of that one